src/flattenHelper.ts

```diff
@@ -5,20 +5,20 @@ import { NodeExtensionHelper, NodeCliHelper } from "./nodeHelper";
 
 export class FlattenHelper {
 
-    public static flattenParameter(req: Request, param: Parameter, path: Property[], prefix: string): void {
+    public static flattenParameter(req: Request, param: Parameter, path: Property[], prefix: string, isPayloadFlatten = false): void {
         if (!(param.schema instanceof ObjectSchema))
             throw Error(`Try to flatten non-object schema: param = '${param.language.default.name}', schema= '${param.schema.language.default.name}'`);
 
-        FlattenHelper.flattenPorperties(req, param, param.schema as ObjectSchema, path, prefix);
+        FlattenHelper.flattenPorperties(req, param, param.schema as ObjectSchema, path, prefix, isPayloadFlatten);
         req.updateSignatureParameters();
     }
 
     public static createFlattenedParameterDefaultName(baseProperty: Property, prefix: string): string {
-        return `${prefix}_${baseProperty.language.default.name}`;
+        return `${prefix}${baseProperty.language.default.name}`;
```

Comment:
>}${ [](start = 24, length = 3)

will this cause backward compatibile issue?

Reply:
I also add `_` at caller